packages/SwingSet/src/kernel/kernel.js

```diff
@@ -634,6 +634,58 @@ export default function buildKernel(
     );
   }
 
+  function setKernelVatOption(vatID, option, value) {
+    switch (option) {
+      case 'reapInterval': {
+        if (
+          typeof value === 'number' ||
```

Comment:
maybe add a Nat?

sounds like a helper function to factor out the option checking between here and the early checks in `vat-vat-admin` might be warranted